pkg/utils/util.go

```diff
@@ -150,6 +150,10 @@ const (
 	// of a VolumeGroupSnapshot, and indicates the name of the latter.
 	VolumeGroupSnapshotNameLabel = "groupsnapshot.storage.k8s.io/volumeGroupSnapshotName"
 
+	// VolumeGroupSnapshotHandleLabel is applied to VolumeSnapshotContents that are member
+	// of a VolumeGroupSnapshotContent, and indicates the handle of the latter.
+	VolumeGroupSnapshotHandleLabel = "groupsnapshot.storage.k8s.io/volumeGroupSnapshotHandle"
```

Comment:
Can you also add a comment that we apply this label to inform the sidecar not to call CSI driver to create snapshot if the snapshot belongs to a group.